.github/workflows/job_pytorch_models_tests.yml

```diff
@@ -98,7 +104,9 @@ jobs:
           self-hosted-runner: ${{ contains(inputs.runner, 'aks') }}
 
       - name: Install OpenVINO Python wheels
-        run: python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+        run: |
+          python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+          python3 -m pip install ${INSTALL_DIR}/openvino-tokenizers.whl
```

Comment:
I suppose wheel has different name:
![image](https://github.com/openvinotoolkit/openvino/assets/2566854/51098039-ddfb-46a6-be3d-de195cdc3ddc)

Reply:
Fixed. Let us see directory structure for OV Tokenizers artifacts also.

Reply:
Located after unzip in `/install/openvino_tokenizers-2024.0.0.0-py3-none-manylinux_2_31_x86_64.whl`